### PR TITLE
Orthogroup tree table  perf

### PR DESCRIPTION
**Pull request by dmfalke**
This PR addresses some performance issues with the protein TreeTable on group pages.

### CODE REVIEW

**Comment by dmfalke** [comment]
I still need to address the issue where the page jumps when a filter is applied.

**Review by bobular — COMMENTED** [review]
Just noticed we need to reset the page number whenever the filters change.

**Comment by bobular** [comment]
From ui-infra

Verbose help text should be replaced by one sentence and "read more..." toggle.

Replace "Note" with a warning icon :warning:

**Comment by bobular** [comment]
Run clustal seems greyed out

**Comment by bobular** [comment]
Don't forget the vertical placement of the tree w.r.t. table (it has gone out of sync a bit)

File: packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/GroupRecordClasses.GroupRecordClass.scss [hunk 1]
```diff
@@ -21,5 +21,9 @@
     max-width: unset;
     width: unset;
     overflow-x: unset;
+
+    tr:has(> td.SelectionCell > div > input:checked) > td {
+      background-color: var(--row-hl-bg-color) !important;
+    }
```
Comment (bobular) [inline packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/GroupRecordClasses.GroupRecordClass.scss:27]: Cool

**Review by bobular — APPROVED** [review]
Looks great. Thanks for decluttering my previous attempts.

**Comment by bobular** [comment]
I had another play with `useDeferredValue`.

This approach seems to reduce the checkbox delay considerably (but not entirely)

```
  const [volatileCorePeripheralFilterValue, setCorePeripheralFilterValue] = useState<
    ('core' | 'peripheral')[]
  >([]);
  const corePeripheralFilterValue = useDeferredValue(volatileCorePeripheralFilterValue);
```
and then only use the "volatile" value in the `SelectList`.


The page/table moving around underneath the `SelectList` widget isn't ideal, as you already mentioned.

**Comment by bobular** [comment]
Maybe even this?

```
import { useState, useDeferredValue } from 'react';

export function useDeferredState<T>(initialValue: T): [T, T, React.Dispatch<React.SetStateAction<T>>] {
  const [volatileValue, setVolatileValue] = useState<T>(initialValue);
  const deferredValue = useDeferredValue(volatileValue);
  
  return [volatileValue, deferredValue, setVolatileValue];
}
```

**Comment by bobular** [comment]
Verbiage suggestion:

Currently
> To see a phylogenetic tree please use a filter to display fewer than 1,000 sequences

Could be
> To see a phylogenetic tree please use the filters to restrict to fewer than 1,000 sequences

**Comment by dmfalke** [comment]
@bobular thanks for the feedback and suggestions. I am going to merge this as it currently is. I think we can explore `useDeferredValue` and updating verbiage in a new PR.

**Pull request merged**